src/server/system_services/schemas/nsfs_account_schema.js

```diff
@@ -29,6 +29,9 @@ module.exports = {
         allow_bucket_creation: {
             type: 'boolean',
         },
+        force_md5_etag: {
+            type: 'boolean',
+        },
```

Comment:
Since it is a schema change suggesting to add a test in the file `test_nc_nsfs_account_schema_validation.test.js`.
For example:
https://github.com/noobaa/noobaa-core/blob/d1bf8c465af745dedbff9d3803ec13da344884ba/src/test/unit_tests/jest_tests/test_nc_nsfs_account_schema_validation.test.js#L33-L38